tests/test_gaussian.py

```diff
@@ -27,6 +27,14 @@
 from defaults import BaseTest
 
 
+U = np.array(
+    [
+        [0.83645892 - 0.40533293j, -0.20215326 + 0.30850569j],
+        [-0.23889780 - 0.28101519j, -0.88031770 - 0.29832709j],
+    ]
+)
+
+
```

Comment:
It might be nice to have test inputs that are  used in several test files organized into a separate file, as then later it will potentially only need to be changed in one place. Although this might change when transitioning entirely to pytest and right now there are only two tests using them.

Reply:
Yep, this is something we should look at in a future PR (port to pytest, and clean up existing tests)